gammapy/maps/axes.py

```diff
@@ -45,6 +45,18 @@ def pix_to_coord(edges, pix, interp="lin"):
     return scale.inverse(interp_fn(pix))
 
 
+PLOT_AXIS_LABEL = {
+    "energy": "Energy",
```

Comment:
This seems very inconsistent. Mixing short mathematical symbols with text. 

Probably we should provide consistent styles here? One with mathematical symbols and one with text?

Reply:
Yes, we should definitely introduce consistency. I personally prefer the text based labels, because of readability and context. We could introduce minimal configuration to let users choose, on the other hand we can just choose one (convention over configuration). Users are always able to overwrite if they don't like it using:
```python
ax = data.plot_something()
ax.set_xlabel("My label")
```

Reply:
Thanks @maxnoe @adonath. It is fine to follow just the text format only. I proposed both as in some cases I wasn't sure about the best way to represent the label. As for example the case of migration: what's the best here? "migration", "True Energy / Energy" or "μ"? If we go to the text format only, I would prefer the notation "True Energy / Energy".
About the options "configuration vs convention", I would vote for convention.